app/src/main/res/values-af/strings.xml

```diff
@@ -2,36 +2,24 @@
 <resources>
   <string name="menu_help">Hulp</string>
   <string name="menu_home">Tuisblad</string>
-  <string name="menu_forward">Verder</string>
```

Comment:
Non en strings.xmls get updated automatically. Its not normally a good idea to manually edit them. Was this done by the tool?

Reply:
Yes. `Analyze` -> `Inspect Code`. The unused resources were removed by the Android Studio itself.

Reply:
Cool can you rebase and I will be able to merge.

Reply:
Done. All checks passed as well. Please verify.

Reply:
I don't think this has been rebased properly. You can forcepush to your branch to pick only the commits you want against master so you arn't including other peoples commits in this.

Reply:
@mhutti1 Can you please elaborate what you meant to say, because git website defines rebasing as `you can take all the changes that were committed on one branch and replay them on another one`. So, the commits of other people will be included to the best of my knowledge. Please guide me about how to not include other people's commits.

Reply:
Commits like 8626ed4 should not be in the change log or files changed but is because of a merge. What I would do is force push your changes ontop of origin/master to this branch.

Reply:
@brijeshshah13 
Let's say your master branch's head was at A when you created a new branch ImproveAPK for this PR. Now you made changes in that branch and now the ImproveAPK branch is at C I.e. A -> C.
In the meantime someone else pushed to the master and now it is at B i.e. A -> B.
What rebasing master in your other branch would do is make it A -> B -> C,  while your master branch remains A -> B.
Hence the changes that should be shown in the change log should only be C i.e. only the ones made by you.
Solution has already been provided above by @mhutti1

Reply:
@mhutti1 @srv-twry Alright. I got it now. Will close this PR and raise a new one with a fresh branch so that it doesn't become messy.

Reply:
@mhutti1 @srv-twry Thank you for your guidance. Now I understood it. Although I think it will better that I create a new PR and a fresh branch to incorporate the changes as my master is out of sync with the remote master and it will take a life time to resolve all the conflicts.